cmd/state/autoupdate.go

```diff
@@ -94,6 +94,10 @@ func autoUpdate(svc *model.SvcModel, args []string, cfg *config.Instance, an ana
 		return false, locale.WrapError(err, locale.T("auto_update_failed"))
 	}
 
+	an.EventWithLabel(anaConst.CatUpdates, anaConst.ActUpdateInstall, anaConst.UpdateLabelSuccess, &dimensions.Values{
+		TargetVersion: ptr.To(avUpdate.Version),
+	})
```

Comment:
The install action should only be invoked by the installer. You are signaling that the autoupdate was successful here, but the event is intended for the installer being successful, regardless of autoupdate.

Reply:
Okay, I've moved all install/success failure analytics into the installer itself.